app/src/main/java/net/skyscanner/backpack/demo/compose/BpkSliderStory.kt

```diff
@@ -52,7 +51,8 @@ fun BpkSliderStory() {
 
 @Composable
 fun RangeSliderSample() {
-  BpkText(text = stringResource(R.string.slider_range), style = TextStyle(fontSize = BpkTheme.typography.label2.fontSize))
+  BpkText(text = stringResource(R.string.slider_range))
```

Comment:
Getting closer! You still need to set the style onto the `BpkText` composable - maybe the docs help https://backpack.github.io/components/text/?platform=compose